tests/components/radarr/test_calendar.py

```diff
@@ -22,7 +21,7 @@ async def test_calendar(
     """Test for successfully setting up the Radarr platform."""
     freezer.move_to("2021-12-02 00:00:00-08:00")
     entry = await setup_integration(hass, aioclient_mock)
-    coordinator: DataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]["calendar"]
+    coordinator: DataUpdateCoordinator = entry.runtime_data.calendar
```

Comment:
We should not access internals during tests. We should let the coordinator trigger itself

Reply:
#119821